### PR TITLE
Switch to GitHub v4 API for the StatusController

### PR DESCRIPTION
**Pull request by machour**
Switched status/index to the GraphQL API by updating `knplabs/github-api` to `^2.11`.

* We're down from 85 REST queries (approx 20 seconds) to 1 GraphQL query (2 seconds).
* Fixes `yiisoft/yii2` showing 3.0.0-alpha1 
* Uses releases to consolidate results if found

Aside from the performance optimization, this will allow you to fetch more information, like number of open issues / pull requests, etc.

Generated table:

Repository | Latest | No Release For | Status | Diff
-- | -- | -- | -- | --
yiisoft/active-record |   |   |   |  
yiisoft/cache |   |   |   |  
yiisoft/db |   |   |   |  
yiisoft/db-elasticsearch |   |   |   |  
yiisoft/db-mongodb |   |   |   |  
yiisoft/db-mssql |   |   |   |  
yiisoft/db-mysql |   |   |   |  
yiisoft/db-oracle |   |   |   |  
yiisoft/db-pgsql |   |   |   |  
yiisoft/db-redis |   |   |   |  
yiisoft/db-sphinx |   |   |   |  
yiisoft/db-sqlite |   |   |   |  
yiisoft/di |   |   |   |  
yiisoft/log |   |   |   |  
yiisoft/rbac |   |   |   |  
yiisoft/yii | 1.1.20 | 232 days |   | 1.1.20...master
yiisoft/yii-auth-client |   |   |   |  
yiisoft/yii-base-api |   |   |   |  
yiisoft/yii-base-web |   |   |   |  
yiisoft/yii-bootstrap3 |   |   |   |  
yiisoft/yii-bootstrap4 |   |   |   |  
yiisoft/yii-captcha |   |   |   |  
yiisoft/yii-console |   |   |   |  
yiisoft/yii-core |   |   |   |  
yiisoft/yii-debug |   |   |   |  
yiisoft/yii-gii |   |   |   |  
yiisoft/yii-http-client |   |   |   |  
yiisoft/yii-jquery |   |   |   |  
yiisoft/yii-masked-input |   |   |   |  
yiisoft/yii-project-template |   |   |   |  
yiisoft/yii-rest |   |   |   |  
yiisoft/yii-swiftmailer |   |   |   |  
yiisoft/yii-twig |   |   |   |  
yiisoft/yii-web |   |   |   |  
yiisoft/yii2 | 2.0.16 | 24 days |   | 2.0.16...master
yiisoft/yii2-apidoc | 2.1.1 | 102 days |   | 2.1.1...master
yiisoft/yii2-app-advanced | 2.0.16 | 24 days |   | 2.0.16...master
yiisoft/yii2-app-basic | 2.0.16 | 24 days |   | 2.0.16...master
yiisoft/yii2-authclient | 2.1.8 | 26 days |   | 2.1.8...master
yiisoft/yii2-bootstrap | 2.0.9 | 25 days |   | 2.0.9...master
yiisoft/yii2-composer | 2.0.7 | 234 days |   | 2.0.7...master
yiisoft/yii2-debug | 2.0.14 | 153 days |   | 2.0.14...master
yiisoft/yii2-docker | 18.12.0 | 80 days |   | 18.12.0...master
yiisoft/yii2-elasticsearch | 2.0.5 | 341 days |   | 2.0.5...master
yiisoft/yii2-faker | 2.0.4 | 369 days |   | 2.0.4...master
yiisoft/yii2-gii | 2.0.8 | 78 days |   | 2.0.8...master
yiisoft/yii2-httpclient | 2.0.7 | 153 days |   | 2.0.7...master
yiisoft/yii2-imagine | 2.1.1 | 367 days |   | 2.1.1...master
yiisoft/yii2-jui | 2.0.7 | 456 days |   | 2.0.7...master
yiisoft/yii2-mongodb | 2.1.7 | 331 days |   | 2.1.7...master
yiisoft/yii2-queue | 2.1.0 | 276 days |   | 2.1.0...master
yiisoft/yii2-redis | 2.0.9 | 153 days |   | 2.0.9...master
yiisoft/yii2-shell | 2.0.2 | 47 days |   | 2.0.2...master
yiisoft/yii2-smarty | 2.0.7 | 305 days |   | 2.0.7...master
yiisoft/yii2-sphinx | 2.0.11 | 153 days |   | 2.0.11...master
yiisoft/yii2-swiftmailer | 2.1.2 | 153 days |   | 2.1.2...master
yiisoft/yii2-twig | 2.2.1 | 153 days | | 2.2.1...master

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -55,8 +55,9 @@
         "abraham/twitteroauth": "~0.7.4",
         "baibaratsky/yii2-rollbar": "^1.7",
         "himiklab/yii2-recaptcha-widget": "^1.2",
-        "helgesverre/spamprotection": "~1.1.0",
-        "yiisoft/yii2-imagine": "~2.1.0"
+        "helgesverre/spamprotection": "1.0.1",
```
Comment (machour) [inline composer.json]: See https://github.com/HelgeSverre/SpamProtection/issues/5

**Comment by machour** [comment]
Forgot to mention, of course I tested that `yii contributors/generate` was still working as expected after upgrading `knplabs/github-api`

**Comment by cebe** [comment]
Thank you!

**Comment by samdark** [comment]
@cebe deployed?

**Comment by machour** [comment]
yep, and it doesn't work on the production server. (works on my work station, and his)
opened a thread on the GitHub forum: https://github.community/t5/GitHub-API-Development-and/Failure-with-a-large-query-on-production-server-works-locally/m-p/19649#M988

**Pull request merged**